lib/media/drm_engine.js

```diff
@@ -76,10 +76,13 @@ shaka.media.DrmEngine = class {
 
     /** @private {function(!shaka.util.Error)} */
     this.onError_ = (err) => {
+      shaka.log.debug('*** onError rejecting all sessions');
       this.allSessionsLoaded_.reject(err);
       playerInterface.onError(err);
     };
 
+    this.licenseRequestCount_ = 0;
```

Comment:
Rather than hard-code the retries. Can you try to make it customizable in format of default argument in a method.

Reply:
Thanks @kenluck2001 this PR is just to help recreate a issue I have seen when rejecting license requests. This code is not for merging.

Reply:
@kenluck2001 Thank you for your review!
@sarge It may not be the best way to use Pull Requests as code snippets. Please submit an issue if you still have a question so that we can keep track.
We'll close this pull request since it's not for merging. Thank you for understanding!

Reply:
@michellezhuogg I appreciate you looking at this. I submitted an issue #3051 and created this to reduce the friction in reproducing the issue. Open to suggestions on how to progress this.